min_dependency_substitutions.rst

```diff
@@ -0,0 +1,28 @@
+.. |NumpyMinVersion| replace:: 1.17.3
```

Comment:
I think these two extra files might be a result of some merge?

Reply:
Oh yeah deleted.